cli/src/commands/migration/rollback.ts

```diff
@@ -0,0 +1,68 @@
+import { Args } from '@oclif/core';
+import { BaseCommand } from '../../base.js';
+import { getBranchDetailsWithPgRoll, isBranchPgRollEnabled } from '../../migrations/pgroll.js';
+import chalk from 'chalk';
+
+export default class MigrationRollback extends BaseCommand<typeof MigrationRollback> {
+  static description = 'Rollback an active migration';
+
+  static examples = [];
+
+  static flags = {
+    ...this.commonFlags,
+    ...this.databaseURLFlag
+  };
+
+  static args = {
+    branch: Args.string({ description: 'The branch to rollback the migration in', required: true })
+  };
+
+  static hidden = true;
+
+  async run() {
+    const { args, flags } = await this.parseCommand();
+
+    const xata = await this.getXataClient();
+    const { workspace, region, database, branch } = await this.getParsedDatabaseURLWithBranch(
+      flags.db,
+      args.branch,
+      true
+    );
+
+    const details = await getBranchDetailsWithPgRoll(xata, { workspace, region, database, branch });
+
+    if (!isBranchPgRollEnabled(details)) {
+      this.error(`${chalk.gray('xata migration')} commands are only supported in Postgres enabled databases`);
+    }
+
+    const commonParams = {
+      region,
+      workspace,
+      dbBranchName: `${database}:${branch}`
+    };
+
+    const migrationJobStatus = await xata.api.migrations.getBranchMigrationJobStatus({
+      pathParams: {
+        ...commonParams
+      }
+    });
+
+    const isActiveMigration = migrationJobStatus.status === 'completed' && migrationJobStatus.type === 'start';
```

Comment:
Note: this is not 100% reliable, if for whatever reason a duplicate migration was submitted, the last migration status can be failed (see example)

`mig_job_cqdp2h97d1aiu327mf2g start failed failed 2024-07-20T10:28:22.45296Z a migration for schema "bb_8dd2korbmd52jfkbmbpfud6ass_d5mrcj" is already in progress`

Yet a migration can be active (the second last migration in this case will be active)

For this to be reliable, we need to ensure that UI, CLI can't call start a job when a migration is running AND any direct API access can't start it either.

That means, either backend needs to reject the job when a migration is already running or we need a new API to reliably detect a running migration instead of trying to find it from the last job status as an alternative we can use the migration history API and find any active migration.